src/components/entity/ha-chart-base.html

```diff
@@ -0,0 +1,182 @@
+<link rel='import' href='../../../bower_components/polymer/polymer.html'>
+
+<dom-module id="ha-chart-base">
+  <link rel="import" type="css" href="../../../bower_components/billboard.js/dist/billboard.min.css">
+  <style>
+     :host {
+      display: block;
+      /* @apply --my-custom-chart-istic-mixin; */
+    }
+  </style>
+  <template>
+    <div id="chartTarget"></div>
+  </template>
+</dom-module>
+<script src="../../../bower_components/d3/d3.js"></script>
+<script src="../../../bower_components/d3-gtimeline/dist/d3-gtimeline.js"></script>
+<script src="../../../bower_components/billboard.js/dist/billboard.js"></script>
+<script>
+/* global d3 bb */
+
+(function () {
+  if (customElements.get('ha-chart-base')) return;
+  /**
+   * `billboard-charts`
+   * Polymer wrapper around billboard.js charting library
+   *
+   * @customElement
+   * @polymer
+   */
+  class ExtenedChart extends Polymer.Element {
+    get chart() {
+      return this._chart;
+    }
+    static get is() { return 'ha-chart-base'; }
+    static get properties() {
+      return {
+        publish: {
+          type: Boolean,
+          observer: 'onPropsChange'
+        },
+        data: {
+          type: Object,
+          observer: 'onPropsChange'
+        },
+        newData: {
```

Comment:
This makes no sense, why not use `data` ? 

Also, we should never call anything new, it is bound to be not the new thing in the future.